dags/resources/stages/parse/table_definitions/benqi/Comptroller_event_ActionPaused.json

```diff
@@ -30,7 +30,7 @@
         "type": "log"
     },
     "table": {
-        "dataset_name": "<INSERT_DATASET_NAME>",
+        "dataset_name": "avalanche_benqi",
```

Comment:
Don't need the chain prefix `avalanche_`. Is this contract deployed on Avalanche?

Reply:
Yea -- think this code might need to go in [this repo](https://github.com/nansen-ai/evmchain-etl-table-definitions/tree/main/parse/table_definitions_avalanche)